ext/js/app/statistics-handler.js

```diff
@@ -0,0 +1,53 @@
+// this class handles all of the setting, updating and wiping of statistics.
+
+export class StatisticsHandler {
+    prepare() {
+        
+        this.initLastUpdate();
+        this.initNumSelects();
+        this.checkIfUpdateRequired();
+    }
+
+    // creates numSelects entry in chrome local storage if doesnt already exist
+    // e.g: if user's first time setting up
+    initNumSelects() {
+        chrome.storage.local.get('numSelects', function(data) {
+            if (typeof data.numSelects === 'undefined') {
+                chrome.storage.local.set({ numSelects: 0 })
```

Comment:
When it comes to storage schema i think we should be a bit careful about forward and backward compatibility. Have you thought about how easy it would be to update or change without losing data?